pdf/model/colorspace.go

```diff
@@ -1413,19 +1413,19 @@ func (this *PdfColorspaceLab) ColorFromFloats(vals []float64) (PdfColor, error)
 
 	// L
 	l := vals[0]
-	if l < 0.0 || l > 1.0 {
+	if l < 0.0 || l > 100.0 {
```

Comment:
Yes, this appears to be correct, however it is inconsistent with our Lab ColorToRGB method, which is assuming some sort of normalized input components.  I will try to identify some test cases for testing.  I rarely encounter Lab colorspace, so might have been relatively less tested.  Unfortunately the standard does not provide any concrete test cases.

Also, the Range parameter can adjust the range of a and b.  I will add that after merging.